src/api/covariates/base.py

```diff
@@ -0,0 +1,12 @@
+from typing import List, Tuple
+
+
+class CovariateRequestError(Exception):
+    pass
+
+
+class BaseCovariate:
+    radius = 0.025  # in km
+
+    def fetch(self, points: List[Tuple[float, float]]) -> List[dict]:
+        raise NotImplementedError()
```

Comment:
Any other properties/methods that should be defined here? `_sqkm_to_sqm` maybe? `num_threads`? Wonder also about the repeated `ThreadPoolExecutor` usages, and whether that should be DRYed up in this file...

Reply:
ok to leave this for now?  Thinking if we do another covariate this way than we do it then, in case it moves to new system